src/Sentry/Sentry.csproj

```diff
@@ -13,10 +14,19 @@
     <DefineConstants>HAS_VALUE_TUPLE;NETSTANDARD;$(AdditionalConstants)</DefineConstants>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetFramework)' == 'net461'">
+  <PropertyGroup Condition="'$(TargetFramework)' != 'net461'">
+    <DefineConstants>HAS_VALUE_TUPLE;$(AdditionalConstants)</DefineConstants>
```

Comment:
I don't follow this change. Did we broke `main` by not marking `net461` as `HAS_VALUE_TUPLE`?

If so can't we just add `HAS_VALUE_TUPLE` to the line below?
`<PropertyGroup Condition="$(TargetFramework.StartsWith('net4'))">`

Reply:
no idea how I manage to change that. Fixed.